x-pack/legacy/plugins/siem/server/lib/detection_engine/routes/schemas.ts

```diff
@@ -213,3 +215,9 @@ export const findRulesSchema = Joi.object({
   }),
   sort_order,
 });
+
+export const setSignalsStatusSchema = Joi.object({
+  signal_ids,
+  query,
```

Comment:
I would use `query: Joi.object();` to make easier on the user and consumer that they don't have to ensure they have valid parseable strings. 

Later if we have a schema validation for queries we can make the object more strict.

Reply:
Sounds like a plan 👍